pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.uid2</groupId>
     <artifactId>uid2-shared</artifactId>
-    <version>2.12.3-7a057c4834</version>
+    <version>2.13.0-SNAPSHOT</version>
```

Comment:
i talked to @thomasm-ttd - please revert this. we shouldn't make changes to this version number in PRs. You can change it for local testing purposes but don't commit it.